common/shlibs

```diff
@@ -3754,7 +3754,7 @@ libdrumstick-alsa.so.1 drumstick-1.1.2_1
 libdrumstick-rt.so.1 drumstick-1.1.2_1
 libnozzle.so.1 libnozzle1-1.11_2
 libmygpo-qt5.so.1 libmygpo-qt-1.1.0_1
-libluv.so.1 libluv-1.30.1.0_1
```

Comment:
You only need to update the shlib in exceptional circumstances or if the soname changes; in this case, it isn't necessary.

Reply:
I thought it was necessary, sorry. I reverted it back.

Reply:
No problem :)